api/app.js

```diff
@@ -2,7 +2,8 @@ const Bugsnag = require('@bugsnag/js');
 const BugsnagPluginExpress = require('@bugsnag/plugin-express');
 const cors = require('cors');
 const express = require('express');
-const routes = require('./routes/index');
+const v1Routes = require('./routes/v1/index');
+const v2Routes = require('./routes/v1.1/index');
```

Comment:
For consistency please can this be called `v1_1Routes` instead of `v2Routes` (also at some point we will probably have v2 routes and so the current naming here would be problem)